flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -131,7 +135,8 @@ public RocksDBKeyedStateBackendBuilder(
 		MetricGroup metricGroup,
 		@Nonnull Collection<KeyedStateHandle> stateHandles,
 		StreamCompressionDecorator keyGroupCompressionDecorator,
-		CloseableRegistry cancelStreamRegistry) {
+		CloseableRegistry cancelStreamRegistry,
```

Comment:
The write batch size seems like an optional parameter, so should not go into the constructor of the builder, but to a setter method.

That way, you also don't need to touch all the tests (which is exactly what the builder should help achieve).

Reply:
done